state/prices/types.ts

```diff
@@ -1,13 +1,32 @@
-import { SynthPrice, PricesMap } from 'sdk/types/prices';
+import Wei from '@synthetixio/wei';
+
+import { SynthPrice, AssetKey } from 'sdk/types/prices';
 import { QueryStatus } from 'state/types';
 
+export type PriceChange = 'up' | 'down' | null;
+export type PriceColorOptions = 'red' | 'green' | 'white';
+
+type PricesInfo<T = Wei> = {
+	price: T;
+	change: PriceChange;
+};
+
+export type PriceColor = {
+	offChain?: PriceColorOptions;
+	onChain?: PriceColorOptions;
+};
+
+export type PriceColors = Record<string, PriceColor>;
```

Comment:
Related to the previous comment I don't think we need this type and the selector it's used for.